libs/pika/runtime_configuration/src/runtime_configuration.cpp

```diff
@@ -125,6 +125,7 @@ namespace pika::util {
 
             // add placeholders for keys to be added by command line handling
             "ignore_process_mask = 0",
+            "override_process_mask = ",
```

Comment:
```suggestion
            "override_process_mask = ${PIKA_OVERRIDE_PROCESS_MASK:}",
```
(or `PIKA_PROCESS_MASK` depending on https://github.com/pika-org/pika/pull/739/files#r1294344840).

Reply:
This would allow setting the mask with
```
PIKA_PROCESS_MASK=$(hwloc-bind --get --taskset) myapp
```
Especially useful if pika doesn't get argc/argv.

Reply:
This would be useful for use in CP2K.

Reply:
#743.